### PR TITLE
Changes to filter question design

### LINKED ISSUE #821: Change filter question label style, combine "Recipient" and "Show me benefits for:" labels
Design:
screen shot 2018-08-07 at 2 26 49 pm

Current app:
screen shot 2018-08-07 at 2 29 36 pm

### PR DESCRIPTION
**Pull request by smcmurtry**
Closes #821

### CODE REVIEW

File: components/radio_selector.js [hunk 1]
```diff
@@ -23,7 +23,13 @@ const styles = theme => ({
   },
   formLabel: {
     lineHeight: "1.5em",
-    marginBottom: "10px"
+    marginBottom: "10px",
+    color: "black !important",
+    fontWeight: "bold"
+  },
+  radioOption: {
```
Comment (maxneuvians) [inline components/radio_selector.js]: 🔥

File: components/radio_selector.js [hunk 2]
```diff
@@ -147,6 +153,7 @@ export class RadioSelector extends React.Component {
                     value={filter_id}
                     control={<Radio />}
                     label={t(filter_id)}
+                    className={classes.radioOption}
```
Comment (maxneuvians) [inline components/radio_selector.js]: 🔥

**Comment by sastels** [comment]
Is the selector boxes supposed to be the same shade as the background or lighter? Also, I guess we’ll need follow up Airtable changes to get the language right?

**Comment by smcmurtry** [comment]
@sastels I talked to Raluca and the profile/needs selector is supposed to be a bit lighter than the background. Also the new language change for the profile question is getting postponed until beta.

**Review by sastels — APPROVED** [review]
LGTM

**Pull request merged**